cliquet/__init__.py

```diff
@@ -327,3 +340,16 @@ def initialize_cliquet(config, version=None, project_name=None):
     # Include cliquet views with the correct api version prefix.
     config.include("cliquet", route_prefix=api_version)
     config.route_prefix = api_version
+
+
+def wrap_app(app, settings):
```

Comment:
The idea is to have this wrap other things, not only new relic, and have the wrappers defined in a configuration, so we need an app wrapper for all the middlewares.